src/languages/python.js

```diff
@@ -358,6 +362,7 @@ export default function(hljs) {
       'gyp',
       'ipython'
     ],
+    unicode: UNICODE_SUPPORTED,
```

Comment:
Lets go with `unicodeRegex: true`...
```suggestion
    unicodeRegex: true,
```

Reply:
And of course compiler will need the name updated there.